tests/integration/routes/profile-test.js

```diff
@@ -98,4 +112,71 @@ getServer(function (error, server) {
 
     group.end()
   })
+
+  test.only('PATCH /session/account/profile', function (group) {
```

Comment:
interestingly, `test.only` causes this test to fail but it doesn’t appear on `npm test`, only on `node tests/integration/routes/profile-test.js` ... kinda tricky :) 

But in your case the tests look good, just remove the `.only()` part